test/Scrumptious.Testing.Service/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Scrumptious.Testing.Service
+{
+    public class Program
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a 'protected' constructor or the 'static' keyword to the class declaration. ([csharpsquid:S1118](https://sonarcloud.io/organizations/mrodsky-github/rules#rule_key=csharpsquid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rodskyproj2&pullRequest=3&issues=AWVI3qENiZQY_aXE0fBV&open=AWVI3qENiZQY_aXE0fBV)

[ISSUE_KEY]: AWVI3qENiZQY_aXE0fBV